packages/utils/src/crypto/sampleGroupElement.ts

```diff
@@ -0,0 +1,29 @@
+import { publicKeyCreate, publicKeyVerify, privateKeyVerify, publicKeyConvert } from 'secp256k1'
+import { randomFillSync } from 'crypto'
+import { SECP256K1 } from './constants'
+
+/**
+ * Generates a secp256k1 keypair used for an
```

Comment:
if the purpose is to create a keypair, why not call it `createKeyPair`?

Reply:
changed the description to make clear that this is not `createKeyPair`